{{cookiecutter.project_slug}}/.github/workflows/pythonapp.yml

```diff
@@ -0,0 +1,65 @@
+# This workflow will install Python dependencies, run tests and lint with multiple version of Python
+name: matchms
+
+on:
```

Comment:
Only the deploy job should be restricted to master branch. The build job should run for any branch or PR.

Reply:
I now have to separate workflows: ci_build.yml and ci_deploy.yml
- build works for all the branches
- It deploys only on release

Reply:
great